### PR TITLE
 UI/UX improvements

### PR DESCRIPTION
**Pull request by naveensingh**
### Changes

 - Using primary color to communicate the on/off state instead of on/off vector drawables
 - Hide brush seek bar and preview while the eyedropper or paint bucket is active

### CODE REVIEW

File: app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt [hunk 1]
```diff
@@ -391,26 +391,44 @@ class MainActivity : SimpleActivity(), CanvasListener {
...
+        my_canvas.toggleBucketFill(isBucketFillOn)
+    }
+
+    private fun updateButtonStates() {
+        hideBrushSettings(isEyeDropperOn || isBucketFillOn)
+
+        arrayOf(
+            eraser to isEraserOn,
+            eye_dropper to isEyeDropperOn,
+            bucket_fill to isBucketFillOn
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt:416]: this is confusing to read, simplify it please

File: app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt [hunk 2]
```diff
@@ -40,6 +40,7 @@ import java.io.ByteArrayOutputStream
 import java.io.File
 import java.io.OutputStream
 
+
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt:43]: avoid adding such double empty lines please

**Comment by tibbi** [comment]
functionality itself seems to be good

File: app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt (hunk 2 above)
Comment (Naveen3Singh) [reply app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt]: sorry, removed them. (was added automatically by the editor when formatting)

File: app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt (hunk 1 above)
Comment (Naveen3Singh) [reply app/src/main/kotlin/com/simplemobiletools/draw/pro/activities/MainActivity.kt]: [done :+1:](https://github.com/SimpleMobileTools/Simple-Draw/pull/248/commits/4957e9a84eab79434df7c191cdeb7801d18524e9)

**Comment by tibbi** [comment]
looks good, thanks

**Pull request merged**